app/models/concerns/form526_rapid_ready_for_decision_concern.rb

```diff
@@ -111,19 +111,27 @@ def update_classification
       form526_submission_id: id
     }
 
+    Rails.logger.info(
+      "classifying 526Submission... id: #{id}, saved_claim_id: #{saved_claim_id}"
```

Comment:
When logging stuff, it is usually better to put variable stuff in the payload, vs interpolated in the message. So for this I would instead do 

```ruby
 Rails.logger.info("classifying 526Submission", id:, saved_claim_id:)
```

Or something like that. When it is done that way then the pertinent fields are more easily parsed out in the logs and actionable on and such (like filterable).

Take this example log message, and how it looks in datadog.

![Screenshot 2023-06-07 at 9 12 15 PM](https://github.com/department-of-veterans-affairs/vets-api/assets/37049625/6c201ba4-9bda-4a8b-aa55-4ad6054b6b7d)

The way you have it here, the id and saved claim id would just be string in the top message part. If you do it in the payload part it shows up in the payload section, which you can filter on and do other things with.

Reply:
I appreciate the example, I'll get that updated